src/psyclone/tests/psyir/nodes/node_test.py

```diff
@@ -669,144 +668,6 @@ def test_scope():
-                                      visibility=Symbol.Visibility.PRIVATE)
-    assert sym.name == "very_private"
-    assert sym.visibility == Symbol.Visibility.PRIVATE
-    assert sym is container.symbol_table.lookup("very_private",
-                                                check_ancestors=False)
-
-
-def test_find_or_create_new_symbol():
-    ''' Check that the Node.find_or_create_symbol() method creates new
-    symbols when appropriate. '''
-    # Create some suitable PSyIR from scratch
-    symbol_table = SymbolTable()
-    symbol_table.add(DataSymbol("tmp", REAL_TYPE))
-    kernel1 = KernelSchedule.create("mod_1", SymbolTable(), [])
-    container = Container.create("container_name", symbol_table,
-                                 [kernel1])
-    xvar = DataSymbol("x", REAL_TYPE)
-    xref = Reference(xvar)
-    assign = Assignment.create(xref, Literal("1.0", REAL_TYPE))
-    kernel1.addchild(assign)
-    assign.parent = kernel1
-    # We have no wildcard imports so there can be no symbol named 'undefined'
-    with pytest.raises(SymbolError) as err:
-        _ = assign.find_or_create_symbol("undefined")
-    assert "No Symbol found for name 'undefined'" in str(err.value)
-    # We should be able to find the 'tmp' symbol in the parent Container
-    sym = assign.find_or_create_symbol("tmp")
-    assert sym.datatype.intrinsic == ScalarType.Intrinsic.REAL
-    # Add a wildcard import to the SymbolTable of the KernelSchedule
-    new_container = ContainerSymbol("some_mod")
-    new_container.wildcard_import = True
-    kernel1.symbol_table.add(new_container)
-    # Symbol not in any container but we do have wildcard imports so we
-    # get a new symbol back
-    new_symbol = assign.find_or_create_symbol("undefined")
-    assert new_symbol.name == "undefined"
-    assert isinstance(new_symbol.interface, UnresolvedInterface)
-    assert type(new_symbol) == Symbol
-    assert "undefined" not in container.symbol_table
-    assert kernel1.symbol_table.lookup("undefined") is new_symbol
-
-
-def test_nemo_find_container_symbol(parser):
-    ''' Check that find_or_create_symbol() works for the NEMO API when the
-    searched-for symbol is declared in the parent module. '''
-    from fparser.common.readfortran import FortranFileReader
-    from psyclone.psyir.nodes import BinaryOperation
-    reader = FortranFileReader(
-        os.path.join(
-            os.path.dirname(os.path.dirname(os.path.dirname(
-                os.path.abspath(__file__)))),
-            "test_files", "gocean1p0", "kernel_with_global_mod.f90"))
-    prog = parser(reader)
-    psy = PSyFactory("nemo", distributed_memory=False).create(prog)
-    # Get a node from the schedule
-    bops = psy._invokes.invoke_list[0].schedule.walk(BinaryOperation)
-    # Use it as the starting point for the search
-    symbol = bops[0].find_or_create_symbol("alpha")
-    assert symbol.datatype.intrinsic == ScalarType.Intrinsic.REAL
-
-
 def test_children_validation():
```

Comment:
pylint says that `ContainerSymbol` is imported but unused.

Reply:
Removed